docs/spec/font.en-US.md

```diff
@@ -0,0 +1,98 @@
+---
+category: Visual
+order: 3
+title: Font
+---
+
+Font is one of the most basic components of a interface design system.
```

Comment:
It is not components. Foundation or basis will be better.

Reply:
Will change to foundations.